pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -193,6 +195,7 @@ enum ProtocolVersion {
 	v13 = 13; // Schema-registry : added avro schema format for json
 	v14 = 14; // Add CommandAuthChallenge and CommandAuthResponse for mutual auth
 	          // Added Key_Shared subscription
+	v15 = 15; // Added sequence_id for SingleMessageMetadata, retrieve the sequence id in batch message.
```

Comment:
We don't need to add a new version here because adding the field on the proto message will not cause any problem if the other end (broker or client) is running an older version.
In protobuf, unknown fields are automatically ignored. 

We just increase the version when adding a new feature that requires a minimum version from the peers, so that we can check pre-emptively.